packages/redux-simple-auth/src/middleware.js

```diff
@@ -120,6 +126,35 @@ export default (config = {}) => {
             return response
           })
         }
+        case INVALIDATE_SESSION: {
+          const state = getState()
+          const authenticatorName = getAuthenticator(state)
+
+          if (authenticatorName) {
+            const authenticator = findAuthenticator(authenticatorName)
+
+            if (!authenticator) {
+              throw new Error(
+                `No authenticator with name \`${action.meta.authenticator}\` ` +
+                  'was found. Be sure you have defined it in the authenticators ' +
+                  'config.'
+              )
+            }
+
+            if (authenticator) {
+              authenticator.invalidate(getSessionData(state)).then(
```

Comment:
Removing the `if` above means we can `return` this promise. See my other comments about returning a value from middleware.